packages/react/package.json

```
@@ -26,8 +26,8 @@
     "!src/**/*.d.ts"
   ],
   "peerDependencies": {
-    "@onfido/castor": "1.0.0",
-    "@onfido/castor-icons": "^1.0.0",
-    "react": "^16.8.0"
+    "@onfido/castor": "^1.0.0",
```

`@onfido/castor` here is being bumped with a repository version bump, to a specific exact version.

If you think that it should use `^`, then a [version bumper](https://github.com/onfido/castor/blob/main/scripts/bump-core-peer-dependency.cjs#L16) should be modified.

If not, I'd say to revert to what it was:

```suggestion
    "@onfido/castor": "1.0.0",
```

---

It should probably allow newer minors and patches too, right?

For example, if you merge dependabot PRs one by one, it'll be annoying because when `castor` and `castor-react` are out of sync `npm` will complain.

I'll update the bumper if you don't mind.